src/jdom/spec.ts

```diff
@@ -197,6 +198,30 @@ export function isReading(pkt: jdspec.PacketInfo) {
     return pkt && pkt.kind == "ro" && pkt.identifier == SystemReg.Reading
 }
 
+const includedRegisters = [
+    SystemReg.Reading,
+    SystemReg.Value,
+    SystemReg.Intensity,
+]
+
+export function isClientRegister(pkt: jdspec.PacketInfo) {
+    return (
+        isRegister(pkt) &&
+        !pkt.lowLevel &&
+        includedRegisters.indexOf(pkt.identifier) > -1
+    )
+}
+
+const ignoredEvents = [SystemEvent.StatusCodeChanged]
+
+export function isClientEvent(pkt: jdspec.PacketInfo) {
```

Comment:
this name is confusing because we have a flag called "client" in the spec and exposed as a field in PacketInfo.

Reply:
fixed